commit df2cf794cede01493f282c99df002ef18de85cd1
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-05-23T19:21:28+01:00 (5 months ago)
    
    OS-7807 smartos-release script needs to reference the branch in platform .tgz
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/tools/smartos-release b/tools/smartos-release
index c35a5b05..bf1a7a51 100755
--- a/tools/smartos-release
+++ b/tools/smartos-release
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -91,7 +91,7 @@ mfind ${SOURCE} -t o | while read OBJECT; do
 done
 
 echo "Updating top level ${SMARTOS} snaplinks"
-mln ${SOURCE}/platform-${TIMESTAMP}.tgz ${SMARTOS}/platform-latest.tgz
+mln ${SOURCE}/platform-${BRANCH}-${TIMESTAMP}.tgz ${SMARTOS}/platform-latest.tgz
 mln ${SOURCE}/smartos-${TIMESTAMP}.iso ${SMARTOS}/smartos-latest.iso
 mln ${SOURCE}/smartos-${TIMESTAMP}-USB.img.gz ${SMARTOS}/smartos-latest-USB.img.gz
 mln ${SOURCE}/smartos-${TIMESTAMP}.vmwarevm.tar.gz ${SMARTOS}/smartos-latest.vmwarevm.tar.gz
